src/components/sliders/attributes.js

```diff
@@ -0,0 +1,255 @@
+    x: {
+        valType: 'number',
+        min: -2,
+        max: 3,
+        dflt: -0.05,
+        role: 'style',
+        description: 'Sets the x position (in normalized coordinates) of the slider.'
+    },
+    pad: extendFlat({}, padAttrs, {
+        description: 'Set the padding of the slider component along each side.'
+    }),
+    xanchor: {
+        valType: 'enumerated',
+        values: ['auto', 'left', 'center', 'right'],
+        dflt: 'left',
+        role: 'info',
+        description: [
+            'Sets the slider\'s horizontal position anchor.',
+            'This anchor binds the `x` position to the *left*, *center*',
+            'or *right* of the range selector.'
+        ].join(' ')
+    },
+    y: {
+        valType: 'number',
+        min: -2,
+        max: 3,
+        dflt: 1,
+        role: 'style',
+        description: 'Sets the y position (in normalized coordinates) of the slider.'
+    },
+    yanchor: {
+        valType: 'enumerated',
+        values: ['auto', 'top', 'middle', 'bottom'],
+        dflt: 'bottom',
+        role: 'info',
+        description: [
+            'Sets the slider\'s vertical position anchor',
+            'This anchor binds the `y` position to the *top*, *middle*',
+            'or *bottom* of the range selector.'
+        ].join(' ')
+    },
+
+    transition: {
+        duration: {
+            valType: 'number',
+            role: 'info',
+            min: 0,
+            dflt: 150,
+            description: 'Sets the duration of the slider transition'
+        },
+        easing: {
+            valType: 'enumerated',
+            values: animationAttrs.transition.easing.values,
+            role: 'info',
+            dflt: 'cubic-in-out',
+            description: 'Sets the easing function of the slider transition'
+        },
+    },
+
+    currentvalue: {
+        visible: {
+            valType: 'boolean',
+            dflt: true,
+            description: [
+                'Shows the currently-selected value above the slider.'
+            ].join(' ')
+        },
+
+        xanchor: {
+            valType: 'enumerated',
+            values: ['left', 'center', 'right'],
+            dflt: 'left',
+            role: 'info',
+            description: [
+                'The alignment of the value readout relative to the length of the slider.'
+            ].join(' ')
+        },
+
+        offset: {
+            valType: 'number',
+            dflt: 10,
+            role: 'info',
+            description: [
+                'The amount of space, in pixels, between the current value label',
+                'and the slider.'
+            ]
+        },
+
+        prefix: {
+            valType: 'string',
+            role: 'info',
+            description: [
+                'When currentvalue.visible is true, this sets the prefix of the lable. If provided,',
+                'it will be joined to the current value with a single space between.'
+            ].join(' ')
+        },
```

Comment:
Solved. Problem exists between screen, chair, and regex. (previous description lacked `.join()` 😬 )